staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4066,6 +4067,8 @@ type TolerationOperator string
 const (
 	TolerationOpExists TolerationOperator = "Exists"
 	TolerationOpEqual  TolerationOperator = "Equal"
+	TolerationOpLt     TolerationOperator = "Lt"
```

Comment:
Given that we define constant values as BumpyCaps (e.g. Equal), should these be spelled out as LessThan and GreaterThan?  Or LT / GT (note caps)?

Is there precedent elsewhere for `Lt` style?

Reply:
Then, are we going to support LessThan/GreaterThan to affinity as well additionally? I don't like one accepts Lt/Gt while the other one does LessThan/GreaterThan.